Lib/fontTools/ttLib/tables/otConverters.py

```diff
@@ -1622,9 +1622,18 @@ def read(self, reader, font, tableDict):
 		regionCount = tableDict["VarRegionCount"]
 		wordCount = tableDict["NumShorts"]
 
+		# https://github.com/fonttools/fonttools/issues/2279
+		longWords = bool(wordCount & 0x8000)
+		wordCount = wordCount & 0x7FFF
+
+		(readBigArray, readSmallArray) = {
+			False: (reader.readShortArray, reader.readInt8Array),
+			True:  (reader.readLongArray,  reader.readShortArray),
+		}[longWords]
```

Comment:
the condition/key being just a bool, is this really faster/more readable than an `if longWords`..?

Reply:
I changed my mind. :)